### ISSUE TITLE
Example with external, 3rd party library

### ISSUE THREAD

**Issue by drewlustro** [mattorchard/eslint-plugin-require-explicit-generics#2]
Hello! 

Thanks for the plugin, although I cannot seem to get it to work.

Config:
```js
 'require-explicit-generics/require-explicit-generics': [
      'error',
      // List your functions here
      [
        'query',
        'ApolloClient.query',
        'ApolloClient<unknown>.query',
        'ApolloClient<TCacheShape>.query',
        'apolloClient.query',
      ],
    ],
```

Code:

```tsx
const response = await apolloClient.query({
    query: QUERY,
    variables: { startupSlug: query.slug },
  });
```

`response` is of type `ApolloQueryResult<any>` because no generic arguments were supplied to `.query`

**Comment by mattorchard** [comment]
Howdy, thank you for creating a ticket!

I have [drafted a fix](https://github.com/mattorchard/eslint-plugin-require-explicit-generics/pull/3) and will hopefully get that merged and deployed later this week, or this weekend.

**Comment by mattorchard** [comment]
@drewlustro I have shipped the fix in version `0.4.1`

Any of each of the following 3 configs now work:
```ts
'require-explicit-generics/require-explicit-generics': [
      'error',
      // List your functions here
      [
        'apolloClient.query',
        '*.query',
        'query', 
      ],
    ],
```

**Comment by drewlustro** [comment]
@mattorchard – thank you 🙏 

I'll give it a shot on Monday.
